### ISSUE TITLE
Escape special characters in attribute values

### ISSUE THREAD

**Issue by mono0x** [k0kubun/hamlit#10]
```haml
%a{title: "'"}
%a{title: "'\""}
```

# Haml

```html
<a title="'"></a>
<a title='&#x0027;"'></a>
```

# Hamlit

```html
<a title='''></a>
<a title=''"'></a>
```

**Comment by eagletmt** [comment]
Not only quotes but other special characters aren't escaped.

## Input
```haml
%a{href: '/search?foo=bar&hoge=<fuga>'}
```

## Expected
```html
<a href='/search?foo=bar&amp;hoge=&lt;fuga&gt;'></a>
```

## Actual
```html
<a href='/search?foo=bar&hoge=<fuga>'></a>
```

**Comment by k0kubun** [comment]
Fixed and released as v0.5.0 :gem:
Thank you :)
